opentibiabr/canary
# fix: support outfit crashes on login

# Description

If you try to log in with support/team outfit (in client 13.32 when open customize character window or login with GM, Community Manager or Customer  Support outfit, this crashes, because starts with mount).

## Behaviour
### **Actual**

1. You are GM or GOD and you login. If you are with support outfit, the client will be crashed.
2. You are GM or GOD, you tried to change your outfit, and opened the customize character successfully, but you selected support outfit with the checkbox *Mount* checked, the client will be crashed either.

## Type of change


## How Has This Been Tested

**Test Configuration**:

  - Server Version: Canary 3.1.2
  - Client: Tibia 13.32
  - Operating System: Windows 11 / Ubuntu 22.04

How to test this functionality:
1. login as GM or GOD, and you are with citizen outfit for example.
3. select any support outfit (GM, Community Manager, or Customer  Support). If the mount checkbox is checked, the client will crash. But if you uncheck this checkbox, and select the support outfit, this will be ok.
4. if you are with a support outfit set. when you log out and log in, the outfit will be changed to citizen (temporary outfit default).
5. You can change your outfit again to any support outfit. **BUT REMEMBER!** You can't open customize character again, it will crash because the check *Mount* will be checked as default from client 13.32. You need to logout and login.

## Checklist
